Mathlib/CategoryTheory/ChosenFiniteProducts.lean

```diff
@@ -274,6 +274,18 @@ theorem braiding_inv_snd {X Y : C} : (β_ X Y).inv ≫ snd _ _ = fst _ _ := by
 
 theorem lift_snd_fst {X Y : C} : lift (snd X Y) (fst X Y) = (β_ X Y).hom := rfl
 
+@[simp]
```

Comment:
(Note: it cannot be `reassoc (attr := simp)` because the composition is only in the RHS.)
```suggestion
@[simp, reassoc]
```